app/views/referee_interface/reference/confirmation.html.erb

```diff
@@ -14,11 +14,10 @@
 <% else %>
   <% content_for :title, t('page_titles.referee.confirmation', full_name: @reference.application_form.full_name) %>
 
-  <div class="govuk-panel govuk-panel--confirmation govuk-!-margin-bottom-8">
-    <h1 class="govuk-panel__title">
-      <%= t('page_titles.referee.confirmation', full_name: @reference.application_form.full_name) %>
-    </h1>
-  </div>
+  <%= govuk_panel(
+    title: t('page_titles.referee.confirmation', full_name: @reference.application_form.full_name),
+    body: nil,
```

Comment:
Tiny nit: the body element still takes up space! 

<img width="666" alt="Screenshot 2021-03-31 at 09 34 56" src="https://user-images.githubusercontent.com/642279/113115520-6bbcec80-9204-11eb-8b88-59c050a47065.png">

Happy to merge this tho — we can fix upstream?

Reply:
Oh yes, bugger. 😕

Reply:
Yeah, we can get a way with it… and when the component is updated we can go back and remove `body: nil`.